packages/@okta/vuepress-site/docs/guides/registration-inline-hook/main/index.md

```diff
@@ -271,6 +269,8 @@ You need to remix your own version of the Okta sample Glitch project and confirm
    * **Authentication Field** = `authorization`
    * **Authorization Secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
 
+   > **Note**: If you want to use OAuth 2.0 to secure your inline hooks, see [Add Authentication method](/docs/guides/common-hook-set-up-steps/nodejs/main/#add-authentication-method).
```

Comment:
See prior comment.

Reply:
This link is a singleton that goes back to the procedure on how to set up OAuth 2.0 for hooks, rather than a link out to the Glitch project that does so.